### DISCUSSION TITLE
edge浏览器安装后不显示按钮，而且B站顶部栏消失，edge和脚本都已经更新到最新

### DISCUSSION THREAD

**Discussion by Luozhe-Socrates** [the1812/Bilibili-Evolved#3796] [提问 / Q&A] [answered]
![98d40ae5d0a2786cff28332ff227b4c](https://user-images.githubusercontent.com/85282104/199710164-8a418c84-f262-4535-aef7-5c9fb268cb52.jpg)

**Comment by the1812** [comment, marked as answer]
#3747
